extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1156,19 +1157,22 @@ export class AmpA4A extends AMP.BaseElement {
 
   /**
    * Shared functionality for cross-domain iframe-based rendering methods.
-   * @param {!Element} iframe Iframe to render.  Should be fully configured
-   * (all attributes set), but not yet attached to DOM.
-   * @param {!string} name
+   * @param {!Object} attributes The attributes of the iframe.
```

Comment:
I'm slightly surprised the type check doesn't complain but I believe this needs to be !Object<string, string> (at least createElementWithAttributes expects it: https://github.com/ampproject/amphtml/blob/master/src/dom.js#L156)